pom.xml

```diff
@@ -0,0 +1,1140 @@
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<groupId>org.apache</groupId>
+		<artifactId>apache</artifactId>
+		<version>20</version>
+	</parent>
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.apache.flink</groupId>
+	<artifactId>flink-connectors</artifactId>
+	<version>1.16-SNAPSHOT</version>
+	<name>Flink : Connectors : </name>
+	<packaging>pom</packaging>
+	<url>https://flink.apache.org</url>
+	<inceptionYear>2022</inceptionYear>
+
+	<licenses>
+		<license>
+			<name>The Apache Software License, Version 2.0</name>
+			<url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+
+	<scm>
+		<url>https://github.com/apache/flink-connector-dynamodb</url>
+		<connection>git@github.com:apache/flink-connector-dynamodb.git</connection>
+		<developerConnection>
+			scm:git:https://gitbox.apache.org/repos/asf/flink-connector-dynamodb.git
+		</developerConnection>
+	</scm>
+
+	<modules>
+		<module>flink-connector-aws-dynamodb</module>
+	</modules>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+		<flink.version>1.16-SNAPSHOT</flink.version>
+		<flink.shaded.version>15.0</flink.shaded.version>
+		<flink.shaded.jackson.version>2.12.4</flink.shaded.jackson.version>
+		<target.java.version>1.8</target.java.version>
+		<maven.compiler.source>${target.java.version}</maven.compiler.source>
+		<maven.compiler.target>${target.java.version}</maven.compiler.target>
+		<scala.version>2.12.7</scala.version>
+		<scala.binary.version>2.12</scala.binary.version>
+
+		<junit4.version>4.13.2</junit4.version>
+		<junit5.version>5.8.1</junit5.version>
+		<assertj.version>3.21.0</assertj.version>
+		<archunit.version>0.22.0</archunit.version>
+		<testcontainers.version>1.16.2</testcontainers.version>
+		<mockito.version>2.21.0</mockito.version>
+		<spotless.version>2.4.2</spotless.version>
+
+		<japicmp.skip>false</japicmp.skip>
+		<japicmp.referenceVersion>1.14.0</japicmp.referenceVersion>
+		<japicmp.outputDir>tools/japicmp-output</japicmp.outputDir>
+
+		<slf4j.version>1.7.36</slf4j.version>
+		<log4j.version>2.17.2</log4j.version>
+
+		<flink.convergence.phase>validate</flink.convergence.phase>
+		<test.randomization.seed/>
+		<test.unit.pattern>**/*Test.*</test.unit.pattern>
+
+		<aws.sdk.version>2.17.249</aws.sdk.version>
+	</properties>
+
+
+
+	<dependencyManagement>
```

Comment:
Let's use the AWS BOM instead of individual dependencies, see https://github.com/apache/flink/pull/20521/files